test/test_podman_baseline.sh

```diff
@@ -60,7 +60,7 @@ podman ps --all
 ########
 # Run ls in redis container, this should work
 ########
-ctrid=$(podman pull registry.access.redhat.com/rhscl/redis-32-rhel7)
+ctrid=$(podman pull redis:4-alpine3.8)
```

Comment:
I'm hesitant to pull this image out as it's been an image that's shown problems in Podman/Buildah before.  Let me think about this.

Reply:
Are any of the currently available RH images multi-arch? If so, I think we should use it because it appears we are only doing an ls in it ?

Reply:
I wasn't able to find any RH multi-arch image publicly available. Maybe I might have not looked in right place. Let me know if you have any specific image in mind that we can use here.